OsmAnd/res/values/strings.xml

```diff
@@ -12,21 +12,21 @@
 -->
 
     <string name="thank_you_for_feedback">Thank you for feedback</string>
-    <string name="poi_cannot_be_found">Node or way cannot be found.</string>
-    <string name="search_no_results_feedback">No search results?\nGive us feedback</string>
+    <string name="poi_cannot_be_found">Could not find node or waypoint.</string>
+    <string name="search_no_results_feedback">No search results?\nProvide feedback</string>
     <string name="release_3_1">
     • Navigation: Fix progress bar, fast swapping of the start and end point of the route\n\n
-    • Map markers: fix turn on/off groups, ability to hide markers from the map\n\n
+    • Map markers: Fix turn on/off groups, ability to hide markers from the map\n\n
     • OSM Edit: Ability to edit tags for non-point objects and ways, fix missing comments on notes, backup of edits\n\n
     • Improve Wikipedia and Wikivoyage parsing, updated files are already available\n\n
-    • Context menu: fix transport shields color in the night mode, fix additional menu sizes\n\n
-    • Boat navigation: support for waterway fairway\n\n
+    • Context menu: Fix transport shields color in the night mode, fix additional menu sizes\n\n
+    • Boat navigation: Support for waterway fairway\n\n
     • Other bug fixes\n\n
     </string>
-    <string name="commiting_way">Committing way…</string>
-    <string name="increase_search_radius_to">Increase search radius to %1$s</string>
-    <string name="send_search_query_description"><![CDATA[We will send your search query: <b>\"%1$s\"</b>, as well as your location.<br/><br/>
-        We do not collect personal information, we only need search data to improve the search algorithm.<br/>]]></string>
+    <string name="commiting_way">Committing waypoint…</string>
+    <string name="increase_search_radius_to">Extend search radius to %1$s</string>
+    <string name="send_search_query_description"><![CDATA[Your search query is sent: <b>\"%1$s\"</b>, as well as your location.<br/><br/>
```

Comment:
This changes the meaning of the message. If I understand the code correctly, the message is shown *before* the search query is sent, so the previous message was more correct.